airbyte-integrations/connectors/destination-oracle/src/test-integration/java/io/airbyte/integrations/destination/oracle/OracleIntegrationTest.java

```diff
@@ -100,12 +100,12 @@ protected JsonNode getFailCheckConfig() {
   }
 
   @Override
-  protected boolean implementsBasicNormalization() {
-    return false;
+  protected boolean implementsNamespaces() {
+    return true;
   }
 
   @Override
-  protected boolean implementsNamespaces() {
+  protected boolean supportsDBT() {
     return true;
```

Comment:
this should be false at the moment right?

Reply:
nope. this support DBT since it supports SQL